pkgs/development/libraries/ffmpeg-full/default.nix

```diff
@@ -365,6 +369,7 @@ stdenv.mkDerivation rec {
     (enableFeature (libmysofa != null) "libmysofa")
     #(enableFeature (libnut != null) "libnut")
     (enableFeature (libopus != null) "libopus")
+    (lib.optional (minVer "5.0" version && libplacebo != null && vulkan-headers != null) "--enable-libplacebo")
```

Comment:
`enableFeature` wasn't viable here as it produces either `--disable-libplacebo` or `--enable-libplacebo` depending on the condition, meanwhile this feature flag does not exist at all before 5.0, making the configure script error out due to unknown flag.

Reply:
The null checks shouldn't be needed, I think.

Reply:
Reworked this line a bit.

What do you mean by "shouldn't be needed"? I'm trying to follow the established pattern, it seems that these checks exist to disable features when:
* The package is unavailable
* The feature is undesired (override e.g. `ffmpeg-full.override { libX11 = null; }`).